src/qml/TransformedPositionSource.qml

```diff
@@ -1,42 +1,102 @@
 import QtQuick 2.12
 import QtPositioning 5.3
-
 import org.qfield 1.0
 import org.qgis 1.0
 import Utils 1.0
+//import QtBluetooth 5.15
```

Comment:
Still needed?

Reply:
Let me make one more test. If there are no trouble with it, I'll use it for the connected state. Otherwise I'll remove it.

Reply:
Okay, it works now with the latest Qt :-) Now we can use it and avoid having the "handmade" connected state. Maybe you can have a final look at my last two commits and then we can merge :tada: